pkg/commands/hosts/list.go

```diff
@@ -28,7 +28,9 @@ Example:
 Formatting:
   The 'hosts list' command supports the following formatting
   options (via the CLI global --format flag):
-    - pretty (default)`
+    - pretty (default)
+		- yaml
```

Comment:
this formatting looks weird, but I don't know if thats just how github renders it. when built and run on my machine it looks like this help text is formatted correctly, so I guess its all good.